docs/src/main/asciidoc/telemetry-opentracing-to-otel-tutorial.adoc

```diff
@@ -0,0 +1,504 @@
+
+It is now time to migrate your application to OpenTelemetry tracing if you haven’t done it yet.
+
+If you need to migrate to Quarkus 2.16.x please beware that configuration properties are different and you should check the older Quarkus OpenTelemetry guide version, https://quarkus.io/version/2.16/guides/opentelemetry#configuration-reference[here].
+
+== Prerequisites
+
+include::{includes}/prerequisites.adoc[]
+
+== Summary
+
+The demo has 5 parts. Please read the summary and then jump to the section that best fits your use case.
+
+1 - The *starting point* presents the quickstart app that uses OpenTracing
+
+2 - The first part is good for anyone performing a *big bang change* of OpenTracing when you don't have any manual instrumentation
+
+3 - This is the *big bang replacement* of OpenTracing when you have manually instrumented the code. We explain the main differences between OpenTracing and OpenTelemetry
+
+4 - The last part uses the *OpenTracing shim*. This is useful if you have a large application with manually instrumented the code. It can help performing the migration step by step because it allows the use of the legacy OpenTracing API on top of new OpenTelemetry API
+
+5 - Conclusion and additional resources
+
+The tasks described bellow fall into 3 categories:
+
+* Dependencies
+* Configuration
+* Code
+
+[[starting-point]]
+== Starting point
+
+The base project was created with Quarkus code generator. At some point that generator will not be able to create projects by using the legacy `quarkus-smallrye-opentracing` extension anymore. For that reason, you can checkout this demo project from GitHub:
+
+
+=== Checkout the existing legacy project
+
+[source,bash]
+----
+git clone git@github.com:brunobat/opentracing-quickstart.git
+----
+
+For convenience, https://github.com/brunobat/opentracing-quickstart[the repository] containing the app to migrate, includes several branches with commits mimicking the migration steps described in this guide. You can checkout the `main` branch to start from the beginning.
+
+=== Generate the legacy project
+
+Create the legacy project by using the https://code.quarkus.io/[Quarkus code generator] with the `resteasy-reactive` and `quarkus-smallrye-opentracing` extensions, or by executing the following command:
+
+:create-app-artifact-id: opentracing-quickstart
+:create-app-extensions: resteasy-reactive,quarkus-smallrye-opentracing
+:create-app-code:
+include::{includes}/devtools/create-app.adoc[]
+
+This command generates the Maven project and imports the `smallrye-opentracing` extension, which
+includes the OpenTracing support and the default https://www.jaegertracing.io/[Jaeger] tracer.
```

Comment:
Could you try to be a bit more consistent with the rest of the guides here? Because typically you will also have the Gradle instructions there.

You don't need the full verbiage here (typically I don't think you need the POM/build.gradle content) but being more consistent would help.

Also, we usually don't mention the code.quarkus.io alternative.

See https://quarkus.io/guides/validation#creating-the-maven-project

Reply:
done